src/components/oParl/Person.tsx

```diff
@@ -46,6 +47,10 @@ export const Person = ({ data, navigation }: Props) => {
     web
   } = data;
 
+  const faction = membership?.find(
+    (membership) => membership.organization?.classification === 'Fraktion' && !membership.endDate
+  );
```

Comment:
a faction here is this compared to a faction in /PersonPreview.tsx 

```
const faction = membership?.find(
    (membership) => membership.organization?.classification === 'Fraktion' && !membership.endDate
  )?.organization;
```

which confuses me. it could be the organization here as well, right? as that is what is used in `right` prop of the `Row`

Reply:
hm ok i see in `onPress` even more is used from a faction, which is different to the other file.
than maybe do the explicit `.organization` as well in the other file to have no differences in nearly the same?

Reply:
i changed it to be aligned in bit files.